src/pages/cw-multi-test.mdx

```diff
@@ -4,30 +4,34 @@ tags: ["multitest"]
 
 import { Callout } from "nextra/components";
 
+[file an issue]: https://github.com/CosmWasm/cw-multi-test/issues
+
 # Introduction
 
-**`MultiTest`** is a suite of testing tools designed for facilitating multi-contract interactions
+**`MultiTest`** is a suite of testing tools designed to facilitate multi-contract interactions
 within the CosmWasm ecosystem. Its primary focus is on providing developers with a robust framework
-for testing complex contract interactions and several Cosmos modules operations.
+for testing complex smart contract interactions and operations involving various Cosmos modules.
 
 <Callout type="info">
-  MultiTest is a **simulator** of the blockchain, allowing tested smart contracts to interact as if
-  they were operating on the real-life blockchain.
+  **`MultiTest`** is a blockchain **SIMULATOR**, allowing tested smart contracts to interact as if
+  they were operating on a real blockchain.
 </Callout>
 
-The most valuable advantage of using **`MultiTest`** is that it allows you to test and debug smart
-contracts with access to Rust source code, eliminating the need to run a complete blockchain node to
-begin designing smart contract functionality.
+The most valuable advantage of using **`MultiTest`** is that it allows testing and debugging of
```

Comment:
You can also write that it is much faster than testing on real blockchain, so it's a good way to catch most of the bugs with the contract.